doc/install/mne_python.rst

```diff
@@ -197,6 +198,17 @@ Python development are:
 
   It should be something like ``C:\Users\user\anaconda3\envs\mne\python.exe``
   (Windows) or ``/Users/user/opt/anaconda3/envs/mne/bin/python`` (macOS).
+
+  If the Spyder console can not start because ``spyder-kernels`` is missing,
+  install the required version in the ``mne`` environment with the following
+  commands in the terminal::
+
+      $ conda activate mne
+      $ conda install spyder-kernels -c conda-forge
```

Comment:
Spyder seems to be very sensitive to the spyder-kernels version (IIRC updating spyder from 5.2.0 to 5.2.2 leads to different spyder-kernels version required), so to avoid problems if user just copy-pastes this command I would change it to something:
```suggestion
      $ conda install spyder-kernels=HERE_EXACT_VERSION -c conda-forge
```
and for more clarity we could explain just below that the version required by spyder can be found here: https://docs.spyder-ide.org/current/troubleshooting/common-illnesses.html#spyder-kernels-not-installed-incompatible
This page is unfortunatelly not up to date with most recent Spyder versions (like 5.2.2), but this issue can be raised on spyder docs repository.

Reply:
LGTM, I'll add it this evening.